### PR TITLE
Basis trading strategy deployment scripts

### PR DESCRIPTION
**Pull request by 0xSachinK**
### Kovan
PerpV2BaseManager: [0x1687eF971F48620bBbD098cBB9D3E32B60D5b1e7](https://kovan-optimistic.etherscan.io/address/0x1687eF971F48620bBbD098cBB9D3E32B60D5b1e7)
DeltaNeutralBasisTradingStrategyExtension: [0xD9a63615f5159f7Abd743E82BCc31c033a595C2E](https://kovan-optimistic.etherscan.io/address/0xD9a63615f5159f7Abd743E82BCc31c033a595C2E)

### Staging Mainnet
PerpV2BaseManager: [0xc210E31Fb7210c39b12511A3733Dc0855A8dB833](https://optimistic.etherscan.io/address/0xc210E31Fb7210c39b12511A3733Dc0855A8dB833)
DeltaNeutralBasisTradingStrategyExtension: [0x9EC0f9187e579b0142Bd0ED2968dc3A720e3F9f2](https://optimistic.etherscan.io/address/0x9EC0f9187e579b0142Bd0ED2968dc3A720e3F9f2)

### Todo
- [x] Update `set-v2-strategies` version once strategy contract is merged in the other repo.

### CODE REVIEW

File: optimism/deployments/utils/dependencies.ts [hunk 1]
```diff
@@ -26,11 +26,18 @@ export default {
...
+      production: ""
+    },
+    69: "0xfA60d10BD76E4Ad2d4A88271f6cd0CA88eDBd08f"
+  },
 
   // PerpV2 virtual tokens
   V_ETH: {
     10: "0x8C835DFaA34e2AE61775e80EE29E2c724c6AE2BB",
-    69: "0x7cE531e940B24EE760e5dDBF8A2E0E359fA04400"
+    69: "0x099FaDE6D13401c4f9742db09ae827E434FB4C30"
```
Comment (cgewecke) [inline optimism/deployments/utils/dependencies.ts:40]: They changed this? (I see it's valid - `vETH2` - just wondering if there's a reason why)

**Review by cgewecke — APPROVED** [review]
Looks good!!

File: optimism/deploy/000_perpSetToken.ts [hunk 2]
```diff
@@ -77,7 +79,9 @@ const func: DeployFunction = trackFinishedStage(CURRENT_STAGE, async function (h
     await usdcTokenInstance.connect(uniSigner).transfer(deployer, usdc(500));
   }
 
-  const perpV2LibAddress = await findDependency(PERPV2_LIBRARY);
+  const perpV2LibraryV2Address = await findDependency(PERPV2_LIBRARY_V2);
+  const positionV2LibAddress = await findDependency(POSITION_V2);
+  const perpV2PositionsLibAddress = await findDependency(PERP_V2_POSITIONS);
   const perpV2LeverageModuleAddress = await findDependency(PERPV2_LEVERAGE_MODULE);
   const slippageIssuanceModuleAddress = await findDependency(SLIPPAGE_ISSUANCE_MODULE);
```
Comment (cgewecke) [inline optimism/deploy/000_perpSetToken.ts:86]: Has there been any discussion about adding a `supplyCapHook` here for production versions? If this product  becomes significant part of the Perp market, doesn't every additional issuance tend to push funding flows in the wrong direction?

Comment (0xSachinK) [reply optimism/deploy/000_perpSetToken.ts:86]: If a supply cap hook is required by IC, they would write one for themselves. We can skip it here.

**Review by cgewecke — APPROVED** [review]
LGTM!

File: optimism/deployments/utils/dependencies.ts (hunk 1 above)
Comment (cgewecke) [reply optimism/deployments/utils/dependencies.ts:40]: 0x8C835DFaA34e2AE61775e80EE29E2c724c6AE2BB looks ok any - getting hit

**Pull request merged**